lib/uploadcare/rails/transformations/image_transformations.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'uploadcare/errors/type_error'
+
+module Uploadcare
+  module Rails
```

Comment:
Is this class Rails-specific?

Reply:
I think, no. But again the question: is it ok if we move it to ruby-gem where this class is not used?

Reply:
I think, this is OK, because the gem is a library.

But we should also ask @dmitry-mukhin.

Reply:
If we (in theory) create an extension for Hanami (just an example), why would we need to duplicate this code...

Reply:
same as for conversion.
this looks like a part of Ruby gem, but may be moved there at later stage.